packages/amplication-client/src/util/analytics.ts

```diff
@@ -1,5 +1,6 @@
 import * as reactTracking from "react-tracking";
 import { REACT_APP_AMPLITUDE_API_KEY } from "../env";
+import { AnalyticsEventNames } from "./analytics-events.types";
 
 export interface Event {
   eventName: string;
```

Comment:
Let's use the new AnalyticsEventNames type here. This will likely have a cascade effect on params types of other function that use the analytics utils.

```suggestion
  eventName: AnalyticsEventNames;
```